assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -168,6 +168,7 @@ function FontControls({ selectedElements, pushUpdate }) {
         {fontWeights && (
           <>
             <DropDown
+              minWidth={140}
```

Comment:
Hmm, if the flex-basis of the font-size stays the same, would reducing the size of the font-weight input actually help with giving more room to the font-size input?

Just tested locally as well and in my case at least, the only thing I see is just more room between the font-weight and the font-size inputs. Should the font-weight perhaps have `flex: 1` instead then to just take the remaining room? This, however, would mean that the list width would differ from the input width...

How it looks locally -- the same width for the font-size, just extra room between:
<img width="295" alt="Screenshot 2020-10-02 at 10 12 26" src="https://user-images.githubusercontent.com/3294597/94897282-94e80480-048f-11eb-9b7c-ba5f5e0fdb27.png">

vs on `main`:
<img width="312" alt="Screenshot 2020-10-02 at 10 15 50" src="https://user-images.githubusercontent.com/3294597/94897436-e85a5280-048f-11eb-851f-b2a7f56d8b6f.png">

Reply:
Good call, I'll switch to flex width's so the dropdown can fill in the space as needed.  When I was testing, I didn't have the editor at full screen so I didn't notice the gap, for me it looked like this:
<img src="https://user-images.githubusercontent.com/40646372/94942957-db3b6480-048b-11eb-972e-8f0e0d65afbf.png" height="300">

But as soon as I went full screen and the panel width increased, then boom, the gap widened 😱 I'll fix.